docs/getting-started.md

```diff
@@ -1,6 +1,6 @@
 # Getting started with balenaEngine
 
-Using balenaEngine (`balena-engine`) should feel very natural to docker users
+Using balenaEngine should feel very natural to Docker users
```

Comment:
The "(`balena-engine`)" term (deleted) had been added in the recent `balenaEngine` renaming as it corresponds to the `docker` command (lower case), but no longer relevant as a capitalised Docker refers to the product:

`balenaEngine` maps to `Docker` (product)
`balena-engine` maps to `docker` (command)